channels/jaicp/src/main/kotlin/com/justai/jaicf/channel/jaicp/reactions/reaction/SmsReaction.kt

```diff
@@ -0,0 +1,21 @@
+package com.justai.jaicf.channel.jaicp.reactions.reaction
+
+import com.justai.jaicf.channel.jaicp.dto.ProviderConfig
+import com.justai.jaicf.channel.jaicp.dto.SmsReply
+import com.justai.jaicf.logging.Reaction
+
+data class SmsReaction(
+    val text: String? = null,
+    val phoneNumber: String? = null,
+    val provider: ProviderConfig? = null,
```

Comment:
I'm not sure that this providerConfig should be saved to reaction as it contains password and sensitive data. I guess it's better to remove it.

Reply:
We need it to set up the sender name (if desired). Mb let's change it to a JsonObject?